.github/workflows/dependencies_update.yml

```diff
@@ -107,15 +130,44 @@ jobs:
           echo "Adding untracked files:"
           git add . --verbose --all
           echo "Committing changes..."
-          git commit --message "🔐 Update dependencies to fix vulnerabilities"
+          git commit --message "$TITLE"
           echo "Pushing branch $branch_name..."
           git push --set-upstream origin "$branch_name"
           echo "Done."
+        env:
+          TITLE: ${{ steps.pr-title.outputs.title }}
 
       - name: Create pull request
+        id: create-pr
         if: ${{ github.ref == 'refs/heads/main' }}
         run: |
-          gh pr create --base main --title "🔐 Update dependencies to fix vulnerabilities" --body "$COMMENT"
+          pr_url=$(gh pr create --base main --title "$TITLE" --body "$COMMENT")
+          echo "Created Pull Request: $pr_url"
+          echo "pr_url=$pr_url" >> "$GITHUB_OUTPUT"
         env:
           COMMENT: ${{ steps.pr-comment.outputs.comment }}
+          TITLE: ${{ steps.pr-title.outputs.title }}
           GH_TOKEN: ${{ github.token }}
+
+      - name: Report failure Status to Slack channel
+        if: ${{ always() }}
```

Comment:
>         if: ${{ always() }}
Does this mean that every run is interpreted as a failure?

Reply:
I will add a comment